Framework/include/QualityControl/QcInfoLogger.h

```diff
@@ -18,6 +18,7 @@
 
 #include <InfoLogger/InfoLogger.hxx>
 #include <InfoLogger/InfoLoggerMacros.hxx>
+#include <boost/property_tree/ptree.hpp>
```

Comment:
Please use `ptree_pwd` in the header and the full `ptree` in the implementation file. We use `QcInfoLogger` everywhere, so it could have a noticeable impact on the build time.

Reply:
correct